docs/ert/reference/configuration/observations.rst

```diff
@@ -23,20 +23,20 @@ to this observation file is declared in the main ERT config file using the
 The observation file is a plain text file, and is in essence built around three
 different classes of observations using the associated keywords:
 
+ - :ref:`HISTORY_OBSERVATION <history_observation>`: For time series of
+   observations for which the observed values can be extracted from
+   WCONPROD or WCONINJE section of ECLIPSE schedule files. Used for easy / quick
```

Comment:
We should say "extracted from the summary files" or something, but mention that the values are propagated by the simulator from the Schedule section of the simulator deck by the simulator. Don't mention Eclipse keywords, as it is not exhaustive.